server/app.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import blogsRouter from './routes/blogs';
+import mongoose from 'mongoose';
+
+// MongoDB Connection
+const dbURI = "mongodb+srv://faustin:faustin@learnnode.e3lxu.mongodb.net/nodeDB?retryWrites=true&w=majority";
+mongoose.connect(dbURI, { useNewUrlParser: true, useUnifiedTopology: true });
+
+const db = mongoose.connection;
```

Comment:
For better code maintainability, I think it's better to put database configuration in a separate config file rather than putting it in the app.js file.

Reply:
Correct.